### PR TITLE
Flex Categories: Add new flex categories for 19-20 versions of curriculum

### PR DESCRIPTION
**Pull request by dmcavoy**
Adding new flex categories that Mike and Hannah requested for CSF and CSP respectively.

### CODE REVIEW

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -947,6 +947,23 @@ en:
...
+    csf_data: 'Data'
+    csf_conditionals: 'Conditionals'
+    csf_sprites: 'Sprites'
+    csf_spritelab: 'Sprite Lab'
+    csf_nested_loops: 'Nested Loops'
+    csf_functions: 'Functions'
+    csf_variables: 'Variables'
+    csf_for_loops: 'For Loops'
+    csf_internet: 'Internet'
+    csf_impacts: 'Impacts of Computing'
```
Comment (davidsbailey) [inline dashboard/config/locales/en.yml:966]: I like that these are namespaced to `csf_` 👍

File: dashboard/config/locales/en.yml [hunk 2]
```diff
@@ -978,6 +995,10 @@ en:
     csp2_1_2018: 'Unit 2: Digital Information'
     csp3_1_2018: 'Unit 3: Intro to Programming'
     csp4_1_2018: 'Unit 4: Big Data and Privacy'
+    csp-variables: 'Variables'
+    csp-conditionals: 'Conditionals'
+    csp-functions: 'Functions'
+    csp-project: 'Project'
```
Comment (davidsbailey) [inline dashboard/config/locales/en.yml]: these should really start with `csp_` (not `csp-`) to follow the convention in this file.

**Review by davidsbailey — APPROVED** [review]
Thanks for adding these! one to-do in comments below.

File: dashboard/config/locales/en.yml (hunk 2 above)
Comment (dmcavoy) [reply dashboard/config/locales/en.yml]: 🤦‍♀️ oops

**Review by davidsbailey — APPROVED** [review]
Looks good!

**Pull request merged**